practitioner-guides/responsiveness.md

```diff
@@ -115,6 +117,7 @@ The following people contributed to this guide:
 * Chan Voong
 * Remy DeCausemaker
 * Luis Cañas Díaz
+* Joachim
```

Comment:
Surname is Noreiko. (I don't have access to edit.)

Thanks!

Reply:
Added! Thank you so much for suggesting this.